.llvm-ci.yml

```diff
@@ -0,0 +1,59 @@
+resources:
+  repositories:
+  - repository: llvm
+    type: github
+    endpoint: LLVMServiceConnection
+    name: llvm/llvm-project
+
+jobs:
+- job: 
+  displayName: LLVM Linux Build
+  pool: VeronaLinuxLLVMBuild
+  strategy:
+    matrix:
+      Clang-8 Debug:
+        CC: clang
+        CXX: clang++
+        CXXFLAGS: -stdlib=libstdc++
+        BuildType: Debug
+        Asan: Off
```

Comment:
Is it worth turning Asan on?  I normally do release and debug + ASan builds for testing (release is fast, if something goes wrong then debug + ASan is quicker at finding failures than just debug).

Reply:
Not really. We only want to build LLVM that has been tested upstream, by selecting the commit hashes that had been proven to pass the kinds of buildbots we care about. This is the job for another script, but essentially, if we find a commit that has already passed ASAN on Linux and Windows, we don't need to test it again here. Same for self-hosting, Arm builds, test-suite, etc.

Reply:
That isn't my goal.  I find ASan LLVM builds *incredibly* useful for catching places where I've made memory management errors by not correctly understanding implicit ownership conventions from LLVM APIs *used in my code*.  I'm not expecting to find new LLVM bugs, I'm expecting to find memory-management bugs in our compiler by running the tests with Debug + ASan.

Reply:
I see. Still, why not just build debug+asan and use that for all Verona build types?